src/Symfony/Component/HttpFoundation/ProblemDetailsJsonResponse.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+use Symfony\Component\HttpFoundation\Exception\ProblemDetailsJsonResponseException;
+
+/**
+ * Represents a JSON response with a Problem Details object.
+ *
+ * @author Abdellah Ramadan <ramadanabdel24@gmail.com>
+ */
+class ProblemDetailsJsonResponse extends Response
+{
+    public function __construct(
+        private ?int $status = null,
```

Comment:
what about making this not nullable and default to 500?

Reply:
`500` is the right default. If we (as the app developers) don't know anything about the problem we've encountered, we have to assume it's our fault. That's a 500.

Reply:
Thank you @derrabus 
Yes. I finally went with 500